src/lib/CRSF/CRSF.cpp

```diff
@@ -752,13 +752,14 @@ void ICACHE_RAM_ATTR CRSF::duplex_set_TX()
   #if (GPIO_PIN_RCSIGNAL_TX == GPIO_PIN_RCSIGNAL_RX)
     ESP_ERROR_CHECK(gpio_set_pull_mode((gpio_num_t)GPIO_PIN_RCSIGNAL_TX, GPIO_FLOATING));
     ESP_ERROR_CHECK(gpio_set_pull_mode((gpio_num_t)GPIO_PIN_RCSIGNAL_RX, GPIO_FLOATING));
-    ESP_ERROR_CHECK(gpio_set_level((gpio_num_t)GPIO_PIN_RCSIGNAL_TX, 0));
     ESP_ERROR_CHECK(gpio_set_direction((gpio_num_t)GPIO_PIN_RCSIGNAL_TX, GPIO_MODE_OUTPUT));
```

Comment:
I think this should be perhaps be copied into each section below the set_level lines so the the level is set before the the pins output is enabled. It may not matter in practice because of external pullup/down resistors.

Reply:
Thanks, makes sense.  Made the change and it works well.